### PR TITLE
DOC-3016 add snapshot data graph

### PR DESCRIPTION
**Pull request by andf-crl**
Addresses: DOC-3016

- [`v22.1/ui-replication-dashboard.md`](https://deploy-preview-14477--cockroachdb-docs.netlify.app/docs/v22.1/ui-replication-dashboard#snapshots-data-received): Added new _Snapshot Data Received_ graph to the Replication metrics page in DB Console. Only one metric tracked, so simple graph description. Waiting on #[13968](https://github.com/cockroachdb/docs/pull/13968) for core metric addition.

### CODE REVIEW

**Review by aayushshah15 — APPROVED** [review]
:lgtm: once we address that one comment.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14477#-:-N80NBkF0HDrfsf4WYiC:bd83sms)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @aayushshah15 and @andf-crl)

___
*[`v22.1/ui-replication-dashboard.md` line 102 at r1](https://reviewable.io/reviews/cockroachdb/docs/14477#-N80N5lx2EmmMHcaKlqr:-N80N5lx2EmmMHcaKlqs:b-uhmcc0) ([raw file](https://github.com/cockroachdb/docs/blob/b1b6585716d56d5cac6e411e55aa500a5d6fc53a/v22.1/ui-replication-dashboard.md#L102)):*
> ```Markdown
> DB Console Replica Snapshots Data Received
> 
> The **Snapshots Data Received** graph shows the amount of data received in bytes by each node via [Raft snapshot transfers](architecture/replication-layer.html#snapshots).
> ```

I'd say "rate of data received" as opposed to the "amount".

**Review by aayushshah15 — APPROVED** [review]
:lgtm_strong: 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14477#-:-N80W_6P4PLl2yPUQNe-:b-uwkowd)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @aayushshah15 and @andf-crl)

**Review by MichaelTrestman — APPROVED** [review]
LGTM with a couple of nits.

File: v22.1/ui-replication-dashboard.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Quiescent | The number of replicas that haven't been accessed for a while.
...
 
-Usually the nodes in a [Raft group](architecture/replication-layer.html#raft) stay synchronized by following along the log message by message.  However, if a node is far enough behind the log (e.g., if it was offline or is a new node getting up to speed), rather than send all the individual messages that changed the range, the cluster can send it a snapshot of the range and it can start following along from there.  Commonly this is done preemptively, when the cluster can predict that a node will need to catch up, but occasionally the Raft protocol itself will request the snapshot.
+Usually the nodes in a [Raft group](architecture/replication-layer.html#raft) stay synchronized by following along with the log message by message.  However, if a node is far enough behind the log (e.g., if it was offline or is a new node getting up to speed), rather than send all the individual messages that changed the range, the cluster can send it a snapshot of the range and it can start following along from there.  Commonly this is done preemptively, when the cluster can predict that a node will need to catch up, but occasionally the Raft protocol itself will request the snapshot.
```
[the remark this answers is not in the document]
Comment (andf-crl) [reply v22.1/ui-replication-dashboard.md:88]: This one's out of scope for this ticket, I just corrected a grammar error.

File: v22.1/ui-replication-dashboard.md [hunk 2]
```diff
@@ -95,6 +95,18 @@ Applied (Learner) | The number of snapshots applied to nodes per second that wer
...
+
+<img src="{{ 'images/v22.1/ui_replica_snapshots_data.png' | relative_url }}" alt="DB Console Replica Snapshots Data Received" style="border:1px solid #eee;max-width:100%" />
+
+The **Snapshots Data Received** graph shows the rate of data received in bytes by each node via [Raft snapshot transfers](architecture/replication-layer.html#snapshots).
+
+On hovering over the graph, the value for the following metric is displayed:
+
+Metric | Description
+-------|------------
+`<node>` | The rate of snapshot data received in bytes per node.
```
[the remark this answers is not in the document]
Comment (andf-crl) [reply v22.1/ui-replication-dashboard.md]: Thanks, you're right! Keeping just `node` though, as it's technically format `(node_num) node_name:node_port` which seems too much here; `node` covers both.

**Comment by andf-crl** [comment]
TYFRs team!! 👍

**Pull request merged**